packages/config/src/layer2s/pepesync.ts

```diff
@@ -0,0 +1,39 @@
+import { ProjectId } from '@l2beat/shared-pure'
+
+import { CONTRACTS, TECHNOLOGY, UPCOMING_RISK_VIEW } from './common'
+import { Layer2 } from './types'
+
+export const pepesync: Layer2 = {
+  isUpcoming: true,
+  type: 'layer2',
+  id: ProjectId('pepesync'),
+  display: {
+    name: 'PepeSync',
+    slug: 'pepesync',
+    description:
+      'PepeSync is a decentralized, EVM Rollup built for the World of Meme Coins. PepeSync is a Layer 2 Launchpad Scaled on Ethureum!',
+    purpose: 'Launchpad,AMM',
+    category: 'Optimistic Rollup',
```

Comment:
Documentation seems to suggest it is intended to be a zkRollup ?

Reply:
Documentation Updated. We forked Docs from Zk  Rollup we are OP Stack.

Thank you